recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ build:
   script_env:
     # Add bootstrapped wheels to `PYTHONPATH` so Python can find them for the build
     - >-
-      PYTHONPATH=  {{- os.sep.join([SRC_DIR|default("."), "boostrap_wheels"]) ~ os.pathsep }}  {{- os.environ.get("PYTHONPATH", "") }}
+      PYTHONPATH= {{- os.sep.join([SRC_DIR|default("."), "boostrap_wheels"]) ~ os.pathsep }} {{- os.environ.get("PYTHONPATH", "") }}
```

Comment:
The linter is warning about the `{{-` syntax. However this is perfectly legal Jinja and used to remove extra blank space

xref: https://github.com/conda-forge/conda-smithy/issues/1837